### ISSUE TITLE
Unexpected token "spread operator" of value "..."

### ISSUE THREAD

**Issue by cuchac** [VincentLanglet/Twig-CS-Fixer#230]
### Rule(s) related to or rule(s) missing

Symfony UX component special spread operator syntax for component arguments

Other Symfony UX Components syntax is working as expected

### Expected behavior
Syntax is OK

### Actual behavior
Error
```
  FATAL   4    |                                          
          5    |    <twig:Alert {{ ...args }}></twig:Alert>                           
          >>   | File is invalid: Unexpected token "spread operator" of value "...".  
          6    | {% endblock %}  
```

**Comment by cuchac** [comment]
If this is really a bug I can try to create a PR. Please point me the file where support should be added.

**Comment by VincentLanglet** [comment]
Hi, this tool does support spread operator https://github.com/VincentLanglet/Twig-CS-Fixer/pull/127

Can you give me an example of file you get the error with ?

I personally parse
```
<twig:Alert {{ ...args }}></twig:Alert>
```
correctly.

**Comment by cuchac** [comment]
Hi, I can reproduce it.

This is my template:
```twig
{% extends "@Development/storybook/preview.html.twig" %}

{% block component %}
   <twig:Alert {{ ...args }}></twig:Alert>
{% endblock %}
```

This is my composer env:
```shell
$ COMPOSER_HOME=/usr/local/lib/composer composer global show
Changed current directory to /usr/local/lib/composer
bamarni/symfony-console-autocomplete v1.5.5  Shell completion for Symfony Console based scripts
nikic/php-parser                     v5.0.2  A PHP parser written in PHP
psr/container                        2.0.2   Common Container Interface (PHP FIG PSR-11)
psr/event-dispatcher                 1.0.0   Standard interfaces for event handling.
psy/psysh                            v0.12.3 An interactive shell for modern PHP.
symfony/console                      v6.4.7  Eases the creation of beautiful and testable command line interfaces
symfony/dependency-injection         v7.0.7  Allows you to standardize and centralize the way objects are constructed in your application
symfony/deprecation-contracts        v3.5.0  A generic function and convention to trigger deprecation notices
symfony/event-dispatcher             v7.0.7  Provides tools that allow your application components to communicate with each other by dispatching events and listening to them
symfony/event-dispatcher-contracts   v3.5.0  Generic abstractions related to dispatching event
symfony/filesystem                   v7.0.7  Provides basic utilities for the filesystem
symfony/finder                       v7.0.7  Finds files and directories via an intuitive fluent interface
symfony/polyfill-ctype               v1.29.0 Symfony polyfill for ctype functions
symfony/polyfill-intl-grapheme       v1.29.0 Symfony polyfill for intl's grapheme_* functions
symfony/polyfill-intl-normalizer     v1.29.0 Symfony polyfill for intl's Normalizer class and related functions
symfony/polyfill-mbstring            v1.29.0 Symfony polyfill for the Mbstring extension
symfony/polyfill-php80               v1.29.0 Symfony polyfill backporting some PHP 8.0+ features to lower PHP versions
symfony/process                      v6.4.7  Executes commands in sub-processes
symfony/property-access              v7.0.7  Provides functions to read and write from/to an object or array using a simple string notation
symfony/property-info                v7.0.7  Extracts information about PHP class' properties using metadata of popular sources
symfony/service-contracts            v3.5.0  Generic abstractions related to writing services
symfony/string                       v7.0.7  Provides an object-oriented API to strings and deals with bytes, UTF-8 code points and grapheme clusters in a unified way
symfony/translation-contracts        v3.5.0  Generic abstractions related to translation
symfony/twig-bridge                  v7.0.7  Provides integration for Twig with various Symfony components
symfony/ux-twig-component            v2.17.0 Twig components for Symfony
symfony/var-dumper                   v7.0.7  Provides mechanisms for walking through any arbitrary PHP variable
symfony/var-exporter                 v7.0.7  Allows exporting any serializable PHP data structure to plain PHP code
twig/twig                            v3.10.2 Twig, the flexible, fast, and secure template language for PHP
vincentlanglet/twig-cs-fixer         2.9.0   A tool to automatically fix Twig code style
webmozart/assert                     1.11.0  Assertions to validate method input/output with nice error messages.
```

**Comment by VincentLanglet** [comment]
Thanks @cuchac,
does https://github.com/VincentLanglet/Twig-CS-Fixer/pull/231 solve the issue ?

**Comment by VincentLanglet** [comment]
Released in 2.10
